docs/src/examples/components/Avatar/Variations/AvatarExampleImageCustomization.shorthand.tsx

```diff
@@ -18,7 +18,7 @@ const AvatarExampleImageCustomizationShorthand = () => (
         // This example does not react to the avatar size variable
         // and otherwise produces bad results when border is applied compared to "normal" image
         <Icon
-          name="user"
+          name="lock"
```

Comment:
Teams theme doesn't have this icon and font icons will not be supported inside of it

Reply:
I am a bit confuse here. We are changing only this icon because Teams' theme doesn't support font icons? What about all other examples like: 'chess rook', 'book' etc. Are we planning to move all examples for now to reflect the Teams' theme icon names?

Reply:
> We are changing only this icon because Teams' theme doesn't support font icons?

This slot in attachment requires an additional styling, so yes.